ReactViewControl/ViewModuleContainer.cs

```diff
@@ -52,17 +59,23 @@ public abstract class ViewModuleContainer : IViewModule {
 
         string[] IViewModule.Events => Events;
 
-        string[] IViewModule.DependencyJsSources => DependencyJsSourcesCache.Value;
+        string[] IViewModule.DependencyJsSources => dependencyJsSourcesDelegate.Invoke();
```

Comment:
Please use `dependencyJsSourcesDelegate?.Invoke() ?? DependencyJsSourcesCache.Value` and remove the constructor assignment of `dependencyJsSourcesDelegate`. Same for CSS